snc.sh

```diff
@@ -133,9 +139,6 @@ function renew_certificates() {
     ${OC} get csr -oname | xargs ${OC} adm certificate approve
 
     delete_operator "daemonset/kubelet-bootstrap-cred-manager" "openshift-machine-config-operator" "k8s-app=kubelet-bootstrap-cred-manager"
-
-    # Remove the cli image which was used for the bootstrap-cred-manager daemonset
-    ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo crictl rmi quay.io/openshift/origin-cli:4.3
```

Comment:
We no longer remove it?

Reply:
@cfergeau no since this image is used for other containers also during the cluster provision so if you delete it here then you will see something like following.
```
+ oc wait --for=delete pod/kubelet-bootstrap-cred-manager-tszwh --timeout=120s -n openshift-machine-config-operator
pod/kubelet-bootstrap-cred-manager-tszwh condition met
+ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i id_rsa_crc core@api.crc.testing -- sudo crictl rmi quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:9e83c13aaaa6d0e8935fc02759478830437457b29b18205533a79e940cb8f9b7
Warning: Permanently added 'api.crc.testing,192.168.126.11' (ECDSA) to the list of known hosts.
time="2020-04-15T15:15:09Z" level=error msg="error of removing image \"quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:9e83c13aaaa6d0e8935fc02759478830437457b29b18205533a79e940cb8f9b7\": rpc error: code = Unknown desc = Image used by fd947e9ecb61ca06d83e5334194fedfd12205add09b2f107e9d9c7445b0eafc6: image is in use by a container"
```